quay_setup.sh

```diff
@@ -8,40 +44,34 @@ oc apply -f cso.yaml
 # add pull secrete for quay images
 oc create secret generic redhat-pull-secret --from-file=".dockerconfigjson=docker_quay.json" --type='kubernetes.io/dockerconfigjson'
 #Quay-operator takes some time to come(especially on crc) up so need to wait before bringing up quayecosystem
-#sleep 5m
 
 #code to wait for quay-operator to come up
 req="No resources found in quay-enterprise namespace."
-while true; do
-  echo "Waiting for QuayEcosystem...."
-  VAR1=$(oc get QuayEcosystem 2>&1)
-  if [[ $VAR1 == $req ]]
-  then
-    echo "QuayEcosystem found"
-    oc apply -f quayecosystem.yaml
-    break
-  fi
-  sleep 1
-done
+com="oc get QuayEcosystem"
+check "\${com}" "\${req}"
```

Comment:
check function should just a command, why again it take some stdout text?
```suggestion
check "oc get QuayEcosystem"
```

Reply:
I didnt get your point.Can you explain?